pytorch_lightning/plugins/apex.py

```diff
@@ -34,6 +34,10 @@ def training_step(self, fx, args):
         output = fx(args)
         return output
 
+    def backward(self, unscaled_loss, optimizer=None, *args, **kwargs):
```

Comment:
Do we want to set optimizer defaults to `None`? Makes sense in native amp as this isn't used, but I think APEX will crash if optimizer is `None`:

https://github.com/NVIDIA/apex/blob/master/apex/amp/handle.py#L17

Reply:
done